hyperopt/tests/test_fmin.py

```diff
@@ -212,3 +214,36 @@ def test_generate_trials_to_calculate(self):
                     )
         assert best['x'] == 0.0
         assert best['y'] == 0.0
+
+
+def test_max_time():
```

Comment:
this probably can be written in a for loop. Ideally with pytest it would be with pytest.mark.parametrize but hyperopt doesn't use pytest yet

Reply:
True, it could be a cycle, but I intentionally keep it like this to make test readable and fast (taking into account that we literally need to wait to test timeout) as possible.